sc/xin2016_pancreas_human.tab.gz.info

```diff
@@ -15,12 +15,12 @@
     "instances": 1492,
     "variables": 39851,
     "source": "<a href='https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81608'>GEO</a>",
-    "url": "https://datasets.biolab.si/sc/xin2016_pancreas_human.pkl.gz",
+    "url": "http://file.biolab.si/scdata/xin2016_pancreas_human.tab.gz",
     "references": [
         "Xin, Y, Kim, J., Okamoto, H., Ni, M., Wei, Y., Adler, C., J. Murphy, A., D. Yancopoulos, Lin, C., Gromada, J. (2016). RNA Sequencing of Single Human Islet Cells Reveals Type 2 Diabetes Genes. Cell Metabolism, 24 (4), 608-615."
     ],
     "seealso": [],
-    "taxid": "10090",
-    "num_of_genes": 39851,
-    "size": 45214358
+    "taxid": "9606",
+    "num_of_genes": 35899,
```

Comment:
Does the number of variables above need to be updated too?

Reply:
Yes, you're right. Nice catch.